docs/generate.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+DOCS_HOME="../../src/main/resources/docs/"
```

Comment:
This is quite dangerous. If you run this script from a different folder it will change, I think.

Can you use something like `ScriptDirectory="$( cd "$( dirname "$0" )" && pwd )"`